cmd/otelcomponentmapping/otelcomponentmapping_status.go

```diff
@@ -0,0 +1,23 @@
+package otelcomponentmapping
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/stackvista/stackstate-cli/cmd/otelmapping"
+	"github.com/stackvista/stackstate-cli/internal/common"
+	"github.com/stackvista/stackstate-cli/internal/di"
+)
+
+// if cli.IsJson() {
+// cli.Printer.PrintJson(
```

Comment:
I think we can get rid of these 😬

Reply:
Oh ._.
I would vouch for having at least an exceptions list for that test :smile: